measurements/wsgi.py

```diff
@@ -1,6 +1,12 @@
+# We assume this is going to be running on gunicorn with the gunicorn config
+# via gunicorn --config python:measurement.gunicorn_config
 from werkzeug.contrib.fixers import ProxyFix
 
 from measurements.app import create_app
 
+from prometheus_flask_exporter.multiprocess import GunicornPrometheusMetrics
+
 application = create_app()
 application.wsgi_app = ProxyFix(application.wsgi_app)
+
+metrics = GunicornPrometheusMetrics(application, group_by='endpoint')
```

Comment:
I assume you will use `metrics` later?

Reply:
There is stuff that can be done by using metrics, but I don't need to currently. I put it inside of a variable so we don't forget that this instance can be used to further customise the metrics endpoint.